defaults/aws_config.yaml

```diff
@@ -7,7 +7,7 @@ user_credentials_path: '~/.ssh/scylla-qa-ec2'
 instance_type_loader: 'c6i.xlarge'
 instance_type_monitor: 't3.large'
 # manual on creating loader AMI's: see docs/new_loader_ami.md
-ami_id_loader: 'scylla-qa-loader-ami-v19'
+ami_id_loader: 'scylla-qa-loader-ami-v20-ubuntu22'
```

Comment:
A few words on how you created this image, which version of c-s is installed on it ?

Reply:
Please don't backport to v14

Not before @soyacz have a chance to try some of the perf runs with it, since it might effect those weekly runs

Reply:
It `will` affect, because everything is newer including the linux core (6.5).
Good case is that effect may be positive... we'll see.

Reply:
I removed v14 backport label.
This is something we don't want to do.
We can consider this in a new branch after reviewing results of both and comparing them.

Reply:
is there a risk `centos` images will disappear from AWS? Shouldn't we drop usage of it everywhere?
New branch will introduce many other changes in one shot making hard to decide if new branch results are good (e.g. introducing EaR for Enterprise, which btw. should be tested).

Reply:
our own images aren't gonna be disappearing

I do agree that we should be testing this on it's own

but we should be creating a new branch more closer to master, for the upcoming work needed (i.e. new features, like tablets)